contract/models/account_analytic_account.py

```diff
@@ -190,3 +196,31 @@ def cron_recurring_create_invoice(self):
             [('recurring_next_date', '<=', fields.date.today()),
              ('recurring_invoices', '=', True)])
         return contracts.recurring_create_invoice()
+
+    @api.multi
+    def action_contract_send(self):
+        self.ensure_one()
+        template = self.env.ref(
+            'contract.email_contract_template',
+            False,
+        )
+        compose_form = self.env.ref('mail.email_compose_message_wizard_form',
+                                    False)
```

Comment:
Why not raising when not found? What happens when it isn't found & the `compose_form.id` is used in the action keys?